.gitignore

```diff
@@ -9,6 +9,7 @@ __pycache__/
 *.pcap
 *.prof
 .ipynb_checkpoints
+/.idea/
```

Comment:
This should go to your ``.git/info/exclude`` or your global gitignore (``config core.excludesFile``).